test/parallel/test-assert.js

```diff
@@ -1015,10 +1023,43 @@ common.expectsError(
   {
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
-    message: 'test'
+    message: 'test',
+    generatedMessage: false
   }
 );
 
+// Do not try to check Node.js modules.
+{
+  const e = new EventEmitter();
+
+  e.on('hello', assert);
+
+  let threw = false;
+  try {
+    e.emit('hello', false);
+  } catch (err) {
+    const frames = err.stack.split('\n');
+    const [, filename, line, column] = frames[1].match(/\((.+):(\d+):(\d+)\)/);
```

Comment:
This might be affected by internal changes of events and assert...can you look for the frame that contains `__filename` instead of using hard-coded offsets?

Reply:
Uh, so this frame is guaranteed to belong to where `EventEmitter.emit` and that should be safe...never mind then, sorry for the noise.